records/config.go

```diff
@@ -73,6 +73,34 @@ type Config struct {
 
 	// EnforceRFC952 will enforce an older, more strict set of rules for DNS labels
 	EnforceRFC952 bool
+
+	// IPSources is the prioritized list of task IP sources
+	IPSources []string // e.g. ["host", "docker", "mesos", "rkt"]
+}
+
+// NewConfig return the default config of the resolver
+func NewConfig() Config {
+	return Config{
+		RefreshSeconds: 60,
+		TTL:            60,
+		Domain:         "mesos",
+		Port:           53,
+		Timeout:        5,
+		SOARname:       "root.ns1.mesos",
+		SOAMname:       "ns1.mesos",
+		SOARefresh:     60,
+		SOARetry:       600,
+		SOAExpire:      86400,
+		SOAMinttl:      60,
+		Resolvers:      []string{"8.8.8.8"},
+		Listener:       "0.0.0.0",
+		HTTPPort:       8123,
+		DNSOn:          true,
+		HTTPOn:         true,
+		ExternalOn:     true,
+		RecurseOn:      true,
+		IPSources:      []string{"docker", "mesos", "host"},
```

Comment:
the default should not include "docker" since Docker, by default, uses IPs that are not cluster routable